acceptance/fixtures/files/retry_script.txt

```diff
@@ -0,0 +1,14 @@
+
+# This script which will fail the first `RETRY_LIMIT` times when run and then
+# will exit successfully on later runs. It will store counter state in a file
+# in the provided `BASEDIR` directory.
+set -x
+
+BASEDIR=${1}
+RETRY_LIMIT=${2}
+
+current=`cat ${BASEDIR}/value.txt || echo '0'`
```

Comment:
I believe this to be true.  I suspect we'll find a number of things (both beaker features and acceptance test calls) that will fail on powershell.  Looking forward (:trollface:) to working through those when we have a solid powershell testing setup to beat against. (If we do already, don't tell me, just let this land and break it to me next week or something).